mm2src/rpc/get_public_key_hash.rs

```diff
@@ -0,0 +1,36 @@
+use common::{mm_ctx::MmArc, mm_error::MmError, HttpStatusCode};
+use derive_more::Display;
+use http::StatusCode;
+use serde_json::Value as Json;
+
+#[derive(Serialize)]
+pub struct GetPublicKeyHashResponse {
+    public_key_hash: String,
+}
+#[derive(Serialize, Display, SerializeErrorType)]
+#[serde(tag = "error_type", content = "error_data")]
+pub enum GetPublicKeyHashError {
+    Internal(String),
```

Comment:
You can provide a more meaningful name to this error than internal. Please also add the displayed message here.

Reply:
I think `GetPublicKeyHashError` is not needed at all since `MmCtx::rmd160()` doesn't return any error.